packages/round-manager/package.json

```diff
@@ -61,7 +61,8 @@
     "lint": "eslint ./src --cache",
     "lint:fix": "eslint ./src --fix --cache",
     "prepare": "cd ../.. && husky install",
-    "lint-staged": "lint-staged"
+    "lint-staged": "lint-staged",
+    "slither": "slither ."
```

Comment:
do we have any solidity files in `round-manager` package? if not, then we don't need the `slither` script here

Reply:
That's a good callout

Reply:
```suggestion
```